src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -52,6 +53,7 @@ public MsgPackLogExporter(GenevaExporterOptions options)
     {
         this.m_tableNameSerializer = new(options, defaultTableName: "Log");
         this.m_exportExceptionStack = options.ExceptionStackExportMode;
+        this.m_eventNameExportMode = options.EventNameExportMode;
```

Comment:
could we store a bool field to indicate if eventname should be exported or not, so that the hot path need less instructions?

Reply:
Updated.